cli.js

```diff
@@ -1135,9 +1135,30 @@ function status(source, json, argv) {
                 moduleMap || sourcePath.name || 'Avo'
               );
 
+              let sourcePathExts = [];
+
+              if (sourcePath.ext === ".js" || sourcePath.ext === ".ts") {
+                sourcePathExts.push("js");
+                sourcePathExts.push("jsx");
+                sourcePathExts.push("ts");
+                sourcePathExts.push("tsx");
+              } else if (sourcePath.ext === ".java" || sourcePath.ext === ".kt") {
+                sourcePathExts.push("java");
+                sourcePathExts.push("kt");
+              } else if (sourcePath.ext === ".m" || sourcePath.ext === ".swift") {
+                sourcePathExts.push("m");
+                sourcePathExts.push("swift");
+              } else {
+                sourcePathExts.push(sourcePath.ext.substring(1));
+              }
+
+              if (argv.verbose) {
+                console.log("Looking in files with extensions:" , sourcePathExts);
+              }
+
               let globs = [
                 new Minimatch(
-                  _.get(source, 'analysis.glob', '**/*' + sourcePath.ext),
+                  _.get(source, 'analysis.glob', '**/*.+(' + sourcePathExts.join("|") + ")"),
```

Comment:
Based on minimatch docs here https://github.com/isaacs/minimatch

Reply:
can you put this link in a comment or something in the code for later referenc?